src/Html/HtmlCssSelectors.fs

```diff
@@ -62,9 +70,8 @@ module internal HtmlCssSelectors =
                 else
                     inQuotes <- true
                     readString acc t
-            | '\\' :: '\'' :: t when inQuotes ->
```

Comment:
@ovatsus It might be a readability issue, is there a better, more idiomatic way to do this? perhaps `not <|` at the beginning or just `not (... )` ?